sanic/headers.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+token, quoted = r"([\w!#$%&'*+\-.^_`|~]+)", r'"((?:[^"]|\\")*)"'
+parameter = re.compile(fr";\s*{token}=(?:{token}|{quoted})", re.ASCII)
+
+
+def parse_options_header(value: str):
```

Comment:
can you add the return type hint while your at it? `-> typing.Tuple[str, dict]`

Reply:
Done.